### DISCUSSION TITLE
I have less Chias today on my wallet

### DISCUSSION THREAD

**Discussion by jatech1126** [Chia-Network/chia-blockchain#11852] [Support]
Why I have less Chias today on my wallet i do not spend any chia. Yesterday i had 15,24 chias and today 14.85

**Comment by loppefaaret** [comment]
Just when you start up the client, the wallet takes a few steps back, and resync the latest data - during this it can appear as if the last few transactions have not been added yet
is this perhaps what you are experiencing ?

**Comment by jatech1126** [comment]
Thanks i fixed the problema

Was a problema with database on my computer

 

De: loppefaaret ***@***.***> 
Enviado el: jueves, 16 de junio de 2022 8:08 a. m.
Para: Chia-Network/chia-blockchain ***@***.***>
CC: jatech1126 ***@***.***>; Author ***@***.***>
Asunto: Re: [Chia-Network/chia-blockchain] I have less Chias today on my wallet (Discussion #11852)

 

Just when you start up the client, the wallet takes a few steps back, and resync the latest data - during this it can appear as if the last few transactions have not been added yet
is this perhaps what you are experiencing ?

—
Reply to this email directly, view it on GitHub <https://github.com/Chia-Network/chia-blockchain/discussions/11852#discussioncomment-2964154> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AZRZDV23PG3UWONCXM5SHEDVPMRMHANCNFSM5YJZHRDA> .
You are receiving this because you authored the thread.  <https://github.com/notifications/beacon/AZRZDVY2WRVN6MO2J4342DTVPMRMHA5CNFSM5YJZHRDKYY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAAWTVOQ.gif> Message ID: ***@***.*** ***@***.***> >



-- 
El software de antivirus Avast ha analizado este correo electrónico en busca de virus.
https://www.avast.com/antivirus
